digits/model/views.py

```diff
@@ -225,6 +233,51 @@ def download(job_id, extension):
     response.headers['Content-Disposition'] = 'attachment; filename=%s_epoch_%s.%s' % (job.id(), epoch, extension)
     return response
 
+@blueprint.route('/<job_id>/delete',
+        methods=['POST'])
+def delete(job_id):
```

Comment:
Thanks! I was not sure what the name should be. I will fix them and since I'm not so familiar with travis-ci,  could not understand the problem yet. There is no problem on my machine.